demo/docbot/ingestion.py

```diff
@@ -8,7 +9,22 @@
 # check out the _bulk python client helper.
 
 # Documents should be ingested from the /data/ dir
+DATA_PATH = os.path.abspath(os.path.join(__file__,  "..", "..","..","data"))
+
+def read_files_from_data() -> list:
```

Comment:
Can we add a docstring to this so that people can know what to expect it should return? Something like the following

```python
"""What the function is expected to do

Args:
    <param_name> (<param type>): <description>
        (default if applicable)

Returns:
    <data type>: <description>
"""